controllers/namespacescope_controller.go

```diff
@@ -1166,6 +1212,86 @@ func (r *NamespaceScopeReconciler) CSVReconcile(req ctrl.Request) (ctrl.Result,
 	return ctrl.Result{RequeueAfter: 180 * time.Second}, nil
 }
 
+func (r *NamespaceScopeReconciler) getWebhooks(csvName string, csvNs string) ([]unstructured.Unstructured, []unstructured.Unstructured, error) {
+	cfg, err := config.GetConfig()
+	if err != nil {
+		klog.Errorf("Failed to get config: %v", err)
+		return nil, nil, err
+	}
+	dynamic := dynamic.NewForConfigOrDie(cfg)
+
+	APIGroup := "admissionregistration.k8s.io"
+	APIVersion := "v1"
+	mKind := "mutatingwebhookconfigurations"
+	vKind := "validatingwebhookconfigurations"
```

Comment:
Are we able to use `r.Client().List` or `r.Reader().List` to obtain the Webhook Configuration?

Here is [an example](https://github.com/IBM/ibm-common-service-operator/pull/1284) from CommonService Operator to exclude resource with certain label